*** AJPA.bst	2013-11-15 08:50:33.988133090 +0100
--- ../../ajpanose/AJPA.bst	2013-11-15 08:50:27.520101017 +0100
***************
*** 64,70 ****
      journal
      key
      month
!     note
      number
      organization
      pages
--- 64,70 ----
      journal
      key
      month
!     %note
      number
      organization
      pages
***************
*** 255,264 ****
  { "in" }
  
  FUNCTION {bbl.pages}
! { "pp." }
  
  FUNCTION {bbl.page}
! { "p." }
  
  FUNCTION {bbl.chapter}
  { "chapter" }
--- 255,264 ----
  { "in" }
  
  FUNCTION {bbl.pages}
! { "p" }
  
  FUNCTION {bbl.page}
! { "p" }
  
  FUNCTION {bbl.chapter}
  { "chapter" }
***************
*** 452,475 ****
      { " " * bbl.pages * }
    if$
  }
! FUNCTION {format.note}
! {
!  note empty$
!     { "" }
!     { note #1 #1 substring$
!       duplicate$ "{" =
!         'skip$
!         { output.state mid.sentence =
!           { "l" }
!           { "u" }
!         if$
!         change.case$
!         }
!       if$
!       note #2 global.max$ substring$ * "note" bibinfo.check
!     }
!   if$
! }
  
  FUNCTION {format.title}
  { title
--- 452,475 ----
      { " " * bbl.pages * }
    if$
  }
! %FUNCTION {format.note}
! %{
! % note empty$
!  %   { "" }
! %    { note #1 #1 substring$
! %      duplicate$ "{" =
! %        'skip$
! %        { output.state mid.sentence =
! %          { "l" }
! %          { "u" }
! %        if$
! %        change.case$
! %       }
! %      if$
! %      note #2 global.max$ substring$ * "note" bibinfo.check
! %    }
! %  if$
! %}
  
  FUNCTION {format.title}
  { title
***************
*** 899,905 ****
      { format.journal.eid }
    if$
    new.block
!   format.note output
    fin.entry
  }
  FUNCTION {book}
--- 899,905 ----
      { format.journal.eid }
    if$
    new.block
!   %format.note output
    fin.entry
  }
  FUNCTION {book}
***************
*** 933,939 ****
    if$
    format.book.pages output
    new.block
!   format.note output
    fin.entry
  }
  FUNCTION {booklet}
--- 933,939 ----
    if$
    format.book.pages output
    new.block
!   %format.note output
    fin.entry
  }
  FUNCTION {booklet}
***************
*** 948,954 ****
    address "address" bibinfo.check output
    format.book.pages output
    new.block
!   format.note output
    fin.entry
  }
  
--- 948,954 ----
    address "address" bibinfo.check output
    format.book.pages output
    new.block
!   %format.note output
    fin.entry
  }
  
***************
*** 977,982 ****
--- 977,983 ----
        format.edition output
        new.sentence
        format.publisher.address output
+       new.block 
      }
      {
        format.chapter.pages "chapter and pages" output.check
***************
*** 986,992 ****
    if$
    format.pages "pages" output.check
    new.block
!   format.note output
    fin.entry
  }
  
--- 987,993 ----
    if$
    format.pages "pages" output.check
    new.block
!   %format.note output
    fin.entry
  }
  
***************
*** 1011,1019 ****
        format.chapter.pages output
      }
    if$
    format.pages "pages" output.check
    new.block
!   format.note output
    fin.entry
  }
  FUNCTION {inproceedings}
--- 1012,1021 ----
        format.chapter.pages output
      }
    if$
+   new.block
    format.pages "pages" output.check
    new.block
!   %format.note output
    fin.entry
  }
  FUNCTION {inproceedings}
***************
*** 1039,1047 ****
      { format.incoll.inproc.crossref output.nonnull
      }
    if$
    format.pages "pages" output.check
    new.block
!   format.note output
    fin.entry
  }
  FUNCTION {conference} { inproceedings }
--- 1041,1050 ----
      { format.incoll.inproc.crossref output.nonnull
      }
    if$
+   new.block
    format.pages "pages" output.check
    new.block
!   %format.note output
    fin.entry
  }
  FUNCTION {conference} { inproceedings }
***************
*** 1057,1063 ****
    address "address" bibinfo.check output
    format.edition output
    new.block
!   format.note output
    fin.entry
  }
  
--- 1060,1066 ----
    address "address" bibinfo.check output
    format.edition output
    new.block
!   %format.note output
    fin.entry
  }
  
***************
*** 1074,1080 ****
    school "school" bibinfo.warn output
    address "address" bibinfo.check output
    new.block
!   format.note output
    fin.entry
  }
  
--- 1077,1083 ----
    school "school" bibinfo.warn output
    address "address" bibinfo.check output
    new.block
!   %format.note output
    fin.entry
  }
  
***************
*** 1088,1094 ****
    new.block
    howpublished "howpublished" bibinfo.check output
    new.block
!   format.note output
    fin.entry
  }
  FUNCTION {phdthesis}
--- 1091,1097 ----
    new.block
    howpublished "howpublished" bibinfo.check output
    new.block
!   %format.note output
    fin.entry
  }
  FUNCTION {phdthesis}
***************
*** 1104,1110 ****
    school "school" bibinfo.warn output
    address "address" bibinfo.check output
    new.block
!   format.note output
    fin.entry
  }
  
--- 1107,1113 ----
    school "school" bibinfo.warn output
    address "address" bibinfo.check output
    new.block
!   %format.note output
    fin.entry
  }
  
***************
*** 1125,1131 ****
      }
    if$
    new.block
!   format.note output
    fin.entry
  }
  
--- 1128,1134 ----
      }
    if$
    new.block
!   %format.note output
    fin.entry
  }
  
***************
*** 1142,1148 ****
    institution "institution" bibinfo.warn output
    address "address" bibinfo.check output
    new.block
!   format.note output
    fin.entry
  }
  
--- 1145,1151 ----
    institution "institution" bibinfo.warn output
    address "address" bibinfo.check output
    new.block
!   %format.note output
    fin.entry
  }
  
***************
*** 1154,1160 ****
    date.block
    format.title "title" output.check
    new.block
!   format.note "note" output.check
    fin.entry
  }
  
--- 1157,1163 ----
    date.block
    format.title "title" output.check
    new.block
!   %format.note "note" output.check
    fin.entry
  }
  
